src/test/java/edu/unc/lib/boxc/migration/cdm/InitializeProjectCommandIT.java

```diff
@@ -186,6 +184,23 @@ public void initNoEnvMappingPathTest() throws Exception {
         assertProjectDirectoryNotCreate();
     }
 
+    @Test
+    public void initNewProjectFromFilesystemTest() throws Exception {
+        String[] initArgs = new String[] {
+                "-w", baseDir.toString(),
+                "--env-config", chompbConfigPath,
+                "init",
+                "-p", "test_file_project",
+                "-s", "files" };
+        executeExpectSuccess(initArgs);
+
+        MigrationProject project = MigrationProjectFactory.loadMigrationProject(baseDir.resolve("test_file_project"));
+        MigrationProjectProperties properties = project.getProjectProperties();
+        assertNewProjectPropertiesSet(properties, "test_file_project", null);
```

Comment:
We should probably just fold this into the test since this is the only place it is used. Otherwise, we should update the name of the method to something like `assertFilesProjectPropertiesSet`